src/openrct2/title/TitleScreen.cpp

```diff
@@ -217,6 +217,9 @@ extern "C"
         openrct2_write_full_version_info(ch, sizeof(buffer) - (ch - buffer));
         gfx_draw_string(dpi, buffer, COLOUR_BLACK, x + 5, y + 5 - 13);
 
+        // Invalidate screen area
+        gfx_set_dirty_blocks(x, y, x + 500, y + 30);
```

Comment:
`x + 500, y + 30`? That's a bit arbitrary, care to explain why these values exactly?